### PR TITLE
Fix CI/CD badge and Logo in Readme

### PR DESCRIPTION
**Pull request by freaz**
## Description
Fixes for readme:

- CI/CD badge
- Superface Logo

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [x] I have read the **CONTRIBUTION_GUIDE** document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # station
 
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/superfaceai/station/CI)
+[![CI / CD](https://github.com/superfaceai/station/actions/workflows/ci_cd.yml/badge.svg)](https://github.com/superfaceai/station/actions/workflows/ci_cd.yml)
 ![TypeScript](https://img.shields.io/badge/%3C%2F%3E-Typescript-blue)
 
 <img src="https://github.com/superfaceai/station/blob/main/docs/LogoGreen.png" alt="superface logo" width="150" height="150">
```
Comment (kysely) [inline README.md:6]: The image should point to _raw_ representation for it to be renderable. You may alter the path if you still wish to move it from `.github` directory.

```suggestion
<img src="https://github.com/superfaceai/station/raw/main/.github/docs/LogoGreen.png" alt="superface logo" width="150" height="150">
```

**Review by kysely — COMMENTED** [review]
Looks good, but I'm afraid the logo won't work when pointing to `blob`.

File: README.md (hunk 1 above)
Comment (freaz) [reply README.md:6]: Works in https://raw.githubusercontent.com/superfaceai/one-sdk-js/main/README.md https://raw.githubusercontent.com/superfaceai/cli/main/README.md

Comment (kysely) [reply README.md:6]: Aah, okay, GitHub does magic to these by changing it to raw automatically. Nevermind then :) 
Just note a logo already exists at `.github/docs/LogoGreen.png` so you might want to use that or remove it in favour of the added one.

**Review by kysely — APPROVED** [review]
:shipit:

File: README.md (hunk 1 above)
Comment (freaz) [reply README.md]: Ah, thanks didn't know. 👍

**Pull request merged**